src/azure-cli/azure/cli/command_modules/storage/_transformers_azure_stack.py

```diff
@@ -0,0 +1,232 @@
+        new_entry['Read'] = str(result[key]['read'])
+        new_entry['Write'] = str(result[key]['write'])
+        new_entry['Delete'] = str(result[key]['delete'])
+        new_entry['RetentionPolicy'] = str(result[key]['retentionPolicy']['days'])
+        new_result.append(new_entry)
+    return new_result
+
+
+def transform_metrics_list_output(result):
+    from collections import OrderedDict
+    new_result = []
+    for service in sorted(result.keys()):
+        service_name = service
+        for interval in sorted(result[service].keys()):
+            item = result[service][interval]
+            new_entry = OrderedDict()
+            new_entry['Service'] = service_name
+            service_name = ''
+            new_entry['Interval'] = str(interval)
+            new_entry['Enabled'] = str(item['enabled'])
+            new_entry['IncludeApis'] = str(item['includeApis'])
+            new_entry['RetentionPolicy'] = str(item['retentionPolicy']['days'])
+            new_result.append(new_entry)
+    return new_result
+
+
+def create_boolean_result_output_transformer(property_name):
+    def _transformer(result):
+        return {property_name: result}
+
+    return _transformer
+
+
+def transform_storage_list_output(result):
+    if getattr(result, 'next_marker', None):
+        logger.warning('Next Marker:')
+        logger.warning(result.next_marker)
+    return list(result)
+
+
+def transform_url(result):
+    """ Ensures the resulting URL string does not contain extra / characters """
+    import re
+    result = re.sub('//', '/', result)
+    result = re.sub('/', '//', result, count=1)
+    return encode_url_path(result)
+
+
+def transform_fs_access_output(result):
+    """ Transform to convert SDK output into a form that is more readily
+    usable by the CLI and tools such as jpterm. """
+
+    new_result = {}
+    useful_keys = ['acl', 'group', 'owner', 'permissions']
+    for key in useful_keys:
+        new_result[key] = result[key]
+    return new_result
+
+
+# TODO: Remove it when SDK is right for file system scenarios
+def transform_fs_public_access_output(result):
+    """ Transform to convert SDK output into a form that is more readily
+    usable by the CLI and tools such as jpterm. """
+
+    if result.public_access == 'blob':
+        result.public_access = 'file'
+    if result.public_access == 'container':
+        result.public_access = 'filesystem'
+    return result
+
+
+# TODO: Remove it when SDK is right for file system scenarios
+def transform_fs_list_public_access_output(result):
+    """ Transform to convert SDK output into a form that is more readily
+    usable by the CLI and tools such as jpterm. """
+
+    new_result = list(result)
+    for i, item in enumerate(new_result):
+        new_result[i] = transform_fs_public_access_output(item)
+    return new_result
+
+
+def transform_metadata(result):
+    return result.metadata
+
+
+# ------------------Track2 Support-----------------------
```

Comment:
We should only put the transform for Track 1 here, right?

Reply:
yes

Reply:
So why not remove this redundant code here?